.travis.yml

```diff
@@ -27,15 +27,42 @@ jobs:
     - stage: test
       env: TOX_ENV=py34
       python: "3.4"
+      before_install: skip
```

Comment:
This is not needed as there is no `before_install` to override.

Reply:
done